build.sbt

```diff
@@ -15,6 +15,11 @@
  * limitations under the License.
  */
 
+// quiet errant sbt linter warnings about unused sbt settings
```

Comment:
Newer versions of sbt adding some built-in linter to validate sbt configurations (likely because it's so hard to know if sbt is doing the right thing). WIthout this it complains the ``classpathTransformerFactories`` aren't actually used. They are clearly being used (the .classpath files do have the correct macro lib dependency), so I think this is just a case where the linter can't check something correctly. It ends up spitting out a bunch of noise, and reccomends this as the solution to quiet it.